images/README.md

```diff
@@ -10,3 +10,22 @@ To add additional applications into the images, open a PR with changes. Follow t
 * Always build from a source to ensure compiler vulnerabilities do not affect the resulting binary
 * Link the binary to a specific version so that it's easier to update when necessary 
 * Build binaries in a separate stage, then copy the resulting binary into the final image to ensure images are small and contain the least number of layers
+
+## Writing image tests
```

Comment:
```suggestion
## Write image tests
```

Reply:
It's continuous work. Should it really be written in simple than continuous?

Reply:
Please read the second bullet point [here](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#headings).

Reply:
Meh.